docker/base/Dockerfile

```diff
@@ -29,6 +29,9 @@ ARG BUILD_GIT_BRANCH
 # Allows docker builds to set the BUILD_GIT_REV
 ARG BUILD_GIT_REV
 
+# Allows docker builds to set the BUILD_TIME
+ARG BUILD_TIME
+
```

Comment:
Should we not be making these changes in `docker/lite` images as well?
cc @frouioui

Reply:
@deepthi, as @fatih changed the `base` image i assumed he was using the `base` Docker image in his environment. But if we always want to have the `build time` configuration I guess we should change it on all Docker images (`base`/`lite`)

Reply:
The `lite` images don't use this arg anywhere. As an example we have these two arguments that are only used in `base`:

```
BUILD_GIT_BRANCH
BUILD_GIT_REV
```

These arguments, just like the one I added are populated via the `tools/build_version_flags.sh` script. But the `lite` images don't use the above args either. So I tried to follow the convention here. I can add them to `lite`, but I'm not sure how useful it'll be.